exercises/06_count_words/06_count_words_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import "testing"
+
+func TestCountWords(t *testing.T) {
+	for _, tc := range []struct {
+		name string
+		str  string
+		want int
+	}{
+		{"empty", "", 0},
```

Comment:
During the lesson we wrote much more tests for this program. Consider adding them as well.